docs/src/components/ui/Banner.tsx

```diff
@@ -38,6 +38,22 @@ const BannerContainer = styled.section`
     font-size: 15px;
     line-height: 12px;
   }
+  :not(.landing &) {
+    ${tmSelectors.hcDark} {
```

Comment:
I know it's a bit late but I wonder can we switch to another API that could be shorter. Something like istead of having three selector for the same properties switch to the following:

```js
...css props

${themed('color: %s', ({ colors }) => colors.neutral0))}
```

where `themed` could be a function that returns three blocks. This approach is better for theme scaling - e.g. if we would need to add or remove one theme we wouldn't need to edit all components.

What do you think?